src/bentoml/_internal/frameworks/utils/onnx.py

```diff
@@ -5,16 +5,15 @@
 from typing import TYPE_CHECKING
 
 from bentoml.exceptions import BentoMLException
-
-from ..types import LazyType
-from .lazy_loader import LazyLoader
+from bentoml._internal.types import LazyType
+from bentoml._internal.utils.lazy_loader import LazyLoader
 
 if TYPE_CHECKING:
     import onnx
     import torch
 
-    from .. import external_typing as ext
-    from ..external_typing import tensorflow as tf_ext  # noqa
+    from bentoml._internal import external_typing as ext
+    from bentoml._internal.external_typing import tensorflow as tf_ext
```

Comment:
can you make this relative import?

Reply:
Do we want to use `...`? Also I vaguely remember that we may prefer absolute import. Are we prefering relative import now?

Reply:
we are doing relative import for all _internal.

Reply:
ok, will change to relative